libs/gisday/platform/ngx/core/src/lib/pages/admin/pages/university/admin-edit-university/admin-detail-university/admin-detail-university.component.ts

```diff
@@ -1,21 +1,29 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { FormBuilder } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
 
 import { UniversityService } from '@tamu-gisc/gisday/platform/ngx/data-access';
 import { University } from '@tamu-gisc/gisday/platform/data-api';
 
 import { BaseAdminDetailComponent } from '../../../base-admin-detail/base-admin-detail.component';
-import { formConfig } from '../../admin-add-university/admin-add-university.component';
+import { formExporter } from '../../admin-add-university/admin-add-university.component';
 
 @Component({
   selector: 'tamu-gisc-admin-detail-university',
   templateUrl: './admin-detail-university.component.html',
   styleUrls: ['./admin-detail-university.component.scss']
 })
-export class AdminDetailUniversityComponent extends BaseAdminDetailComponent<University> {
+export class AdminDetailUniversityComponent extends BaseAdminDetailComponent<University> implements OnInit {
   constructor(private fb1: FormBuilder, private route1: ActivatedRoute, private universityService: UniversityService) {
     super(fb1, route1, universityService);
-    this.formGroup = formConfig;
+    this.form = formExporter();
```

Comment:
Don't intiialize business logic in the constructor unless you need to. `form` does not need to be, I don't think. Can be moved to the OnInit.